pkg/stolon/stolon.go

```diff
@@ -99,6 +99,26 @@ func (c Clusterdata) CheckHealthy() error {
 	return nil
 }
 
+// CheckMinHealthy returns an error if there aren't the minimum number of healthy
+// keepers to perform a failover (healthy keepers >1 excluding the master)
+func (c Clusterdata) CheckMinHealthy() error {
+	healthy := 0
+Loop:
+	for _, db := range c.Dbs {
+		if db.Spec.KeeperUID == c.Master().Spec.KeeperUID {
+			continue Loop
+		}
+		if db.Status.Healthy {
+			healthy++
+		}
+		if healthy > 1 {
+			return nil
+		}
+	}
+
+	return fmt.Errorf("not enough healthy keepers to failover")
```

Comment:
Also this error is out of place. Who said anything about failover in this package?

Reply:
This is true!

Reply:
Done!